Mixpanel/MPABTestDesignerConnection.m

```diff
@@ -179,7 +179,7 @@ - (void)sendMessage:(id<MPABTestDesignerMessage>)message
     NSData *jsonData = [message isKindOfClass:[NSString class]] ? [(NSString *)message dataUsingEncoding:NSUTF8StringEncoding] : message;
 
     NSError *error = nil;
-    id jsonObject = [NSJSONSerialization JSONObjectWithData:jsonData options:0 error:&error];
+    id jsonObject = [NSJSONSerialization JSONObjectWithData:jsonData options:(NSJSONReadingOptions)0 error:&error];
```

Comment:
Just noticed this while reviewing an update of the library in our project. I guess it would be better to use `kNilOptions` here instead of casting. Should I submit a pull request or someone else wants to?